### PR TITLE
109 fix api

### PR DESCRIPTION
**Pull request by Cozloschi**
@vtemian

### CODE REVIEW

File: src/dashboard/tests/api.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.test import TestCase
+from django.test.client import Client
+
+
+class TestApiResponse(TestCase):
+    def test_api_response(self):
+        c = Client()
```
Comment (vtemian) [inline src/dashboard/tests/api.py]: `c` is not that fun. Try
```response = Client().get('/api/google.ro')```

File: src/dashboard/tests/api.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        response = c.get('/api/google.ro')
+        self.assertEqual(response.status_code, 301)
```
Comment (vtemian) [inline src/dashboard/tests/api.py]: Test for response also..not just for redirect

**Comment by Cozloschi** [comment]
@vtemian

**Comment by vtemian** [comment]
lgtm

**Pull request merged**